hardware/plugins/PluginTransports.cpp

```diff
@@ -527,7 +527,7 @@ namespace Plugins {
 
 				m_Socket = new boost::asio::ip::udp::socket(ios, boost::asio::ip::udp::endpoint(boost::asio::ip::udp::v4(), iPort));
 				m_Socket->set_option(boost::asio::ip::udp::socket::reuse_address(true));
-				if ((m_IP.substr(0, 4) >= "224.") && (m_IP.substr(0, 4) <= "239.") || (m_IP.substr(0, 4) == "255."))
+				if (((m_IP.substr(0, 4) >= "224.") && (m_IP.substr(0, 4) <= "239.")) || (m_IP.substr(0, 4) == "255."))
 				{
```

Comment:
If we had a forces format of:

```
if (
	(
		(m_IP.substr(0, 4) >= "224.")
		&& (m_IP.substr(0, 4) <= "239.")
	)
	|| (m_IP.substr(0, 4) == "255.")
   )
   {
   ......
   }

```  

The original author might have seen this coming ;)